src/Core/Session.php

```diff
@@ -123,7 +123,10 @@ public static function setVisitorsContacts()
 				continue;
 			}
 
-			$session->set('remote', [$contact['uid'] => $contact['id']]);
+			/* get+set to append */
+			$remote_value = $session->get('remote', null);
```

Comment:
To avoid a type error if the session variable doesn't exist.
```suggestion
			$remote_value = $session->get('remote', []);
```

Reply:
Thanks.  Very sensible. Done.